setup.py

```diff
@@ -13,27 +13,23 @@
     'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
     'Topic :: Software Development',
     'Topic :: Software Development :: Libraries :: Application Frameworks',
-    'Programming Language :: Python :: 3.4',
-    'Programming Language :: Python :: 3.5',
     'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
     'Framework :: Django',
-    'Framework :: Django :: 1.11',
-    'Framework :: Django :: 2.0',
-    'Framework :: Django :: 2.1',
     'Framework :: Django :: 2.2',
+    'Framework :: Django :: 3.0',
+    'Framework :: Django :: 3.1',
+    'Framework :: Django :: 3.2',
 ]
 
 INSTALL_REQUIREMENTS = [
-    'Django>=1.11,<3.0',
+    'Django>=2.2,<4.0',
     'django-parler>=1.4',
     'django-cms',
 ]
 
-TEST_REQUIRE = [
-    "djangocms_helper",
-    'djangocms-versioning',
-]
-
 setup(
     name='djangocms-alias',
     author='Divio AG',
```

Comment:
While we're changing things in here can we add the maintainer details after the author info;

```
    maintainer='Django CMS Association and contributors',
    maintainer_email='info@django-cms.org',
```

Reply:
Sure, will add it now.